src/Workspaces/Core/Portable/Workspace/Host/PersistentStorage/IPersistentStorageLocationService.cs

```diff
@@ -20,8 +21,13 @@ internal interface IPersistentStorageLocationService : IWorkspaceService
         string? TryGetStorageLocation(Solution solution);
     }
 
+    internal interface IPersistentStorageLocationService2 : IPersistentStorageLocationService
+    {
+        string? TryGetStorageLocation(Workspace workspace, SolutionKey solutionKey);
+    }
```

Comment:
@Therzok This is the new API.  I need this to not break you if you are actually implementing IPersistentStorageLocationService.  However, if you're just subclassing our DefaultPersistentStorageLocationService then we woudln't need this.